time stamp         #时间戳
string cmd         #任务
string subcmd      #交通信号灯、控制状态
path_point[] path  #路径
bool final_path
string task_id


